examples/hellocluster/launch_hellocluster.py

```diff
@@ -1,15 +1,18 @@
 import logging
-import os
 
-from slurmpilot.config import default_cluster_and_partition
+from pathlib import Path
+
+from slurmpilot.config import default_cluster_and_partition, load_config
 from slurmpilot.slurm_wrapper import SlurmWrapper, JobCreationInfo
 from slurmpilot.util import unify
 
 if __name__ == '__main__':
     logging.basicConfig(level=logging.INFO)
-    cluster, partition = default_cluster_and_partition()
+    user_path = Path('slurmpilot/config')
+    config = load_config(user_path=user_path)
+    cluster, partition = default_cluster_and_partition(user_path=user_path)
```

Comment:
Could we avoid setting the `user_path` and use the default one instead?

```suggestion
    config = load_config()
    cluster, partition = default_cluster_and_partition()
```

which would look at `~/slurmpilot/config` by default.